justfile

```
@@ -169,6 +169,7 @@ env-info:
     {{just_executable()}} --version
     rustc --version
     cargo --version
+    c
```

I don't know why this was not committed.

```suggestion
    cargo --list
```